### PR TITLE
Izaak changes

### PR DESCRIPTION
**Pull request by karlhungus**
- [x] fortune `smac`
- [x] YCM
- [ ] fzf + remove command-t?
- [ ] dev (maybe only osx?)
- [ ] [prezto](https://github.com/sorin-ionescu/prezto) [howto](http://linhmtran168.github.io/blog/2013/12/15/ditching-oh-my-zsh-for-prezto/)

### CODE REVIEW

File: lib/deltas/files.sh [hunk 1]
```diff
@@ -36,4 +36,14 @@ apply() {
...
+
+  if osx; then
+    pushd lib/fortune/fortune-mod-smac-0.1-osx/
+  fi
+
+  if linux; then
+    pushd lib/fortune/fortune-mod-smac-0.1/
+  fi
+  make install
+  popd
```
Comment (karlhungus) [inline lib/deltas/files.sh:48]: @pseudomuto is this like... not as intended?

**Comment by karlhungus** [comment]
OOOOOppppsie (ment to open this on my fork)

File: lib/deltas/files.sh (hunk 1 above)
Comment (pseudomuto) [inline lib/deltas/files.sh:48]: You should pipe `push/popd` to `/dev/null` otherwise it'll show crap in the terminal

File: files/.zshrc [hunk 2]
```diff
@@ -19,3 +19,14 @@ if [[ `uname` == "Darwin" ]]; then
...
+if [[ $OSTYPE == darwin* ]]; then
+  /usr/local/bin/fortune smac
+fi
+if [[ $OSTYPE == linux* ]]; then
+  /usr/games/fortune smac
+fi
+
+
+[ -f ~/.fzf.zsh ] && source ~/.fzf.zsh
+[ -f /opt/dev/dev.sh ] && source /opt/dev/dev.sh
```
Comment (pseudomuto) [inline files/.zshrc:32]: This is already in `~/.shellrc.d/sources`

**Pull request closed without merging**